spec/models/student_loans/permitted_parameters_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe StudentLoans::PermittedParameters do
+  let(:claim) { TslrClaim.new }
+  let(:permitted_parameters) { described_class.new(claim) }
+
+  describe "keys" do
+    context "when no fields have come from verify" do
+      it "returns all permitted parameters" do
+        expect(permitted_parameters.keys).to eq(StudentLoans::PermittedParameters::PARAMETERS)
+      end
+    end
+
+    context "when fields have come from verify" do
+      before do
+        claim.verified_fields = ["gender"]
```

Comment:
Should be `payroll_gender`, which explains why the test was passing.

Reply:
hmmm, I thought all these had been updated... Will have another look.